README.md

```diff
@@ -60,6 +60,15 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: [Join #Orbtoberfest](https://circleci-public.github.io/hacktoberfest/#/) and submit your PRs to this [Google Form](https://docs.google.com/forms/d/1-MbzLy0m1eRLjeye9wu76fYkzcX5A3PvgKd09EhVObY/viewform).
 - **Notes**: View full details above. Make pull requests to any open-source "orbs" for CircleCI, or create your own. When complete submit PRs to the Google form to claim swag.
 
+### D
+
+#### **Devfolio**
+
+- **Swag**: Stickers, Limited Edition T-Shirts
+- **Requirements**: Get atleast four pull request merged into any project [react-otp-input](https://github.com/devfolioco/react-otp-input), [devfolio-gatsby-starter](https://github.com/devfolioco/devfolio-gatsby-starter) or [react-intl-tel-input](https://github.com/patw0929/react-intl-tel-input) to win stickers and for four exceptional pull requests get a chance to earn their LIMITED EDITION t-shirts!
```

Comment:
Their blog post lists [4 repos](https://devfolio.co/blog/hacktoberfest-2019-devfolio/) and please double check your spelling here. :wink: